package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tableflip-www",
-  "version": "1.6.0",
```

Comment:
The current version of the site is 1.7.0
I pulled from `origin master` and that is what updated my `package.json` to version 1.7.0
This change is a patch so I'm submitting 1.7.1